src/Test/Unit/Config/Schema/_files/.magento.env.md

```diff
@@ -0,0 +1,20 @@
+# ENV_VARIABLE
```

Comment:
In DevDocs, the heading 1 (single hash) is reserved for the page title. Not sure if that rule should apply here, but if there is a page title, you may want to consider making this a heading 2.

Reply:
Fixed